src/cryptography/hazmat/backends/openssl/decode_asn1.py

```diff
@@ -695,7 +695,7 @@ def _asn1_to_der(backend, asn1_type):
 def _asn1_integer_to_int(backend, asn1_int):
     bn = backend._lib.ASN1_INTEGER_to_BN(asn1_int, backend._ffi.NULL)
     backend.openssl_assert(bn != backend._ffi.NULL)
-    bn = backend._ffi.gc(bn, backend._lib.BN_free)
+    bn = backend._ffi.gc(bn, backend._lib.BN_clear_free)
```

Comment:
I'm pretty sure these are always public, since they're from an X.509 cert -- does that sound right @reaperhulk ?

Reply:
Every point where we utilize this is within X509 and is public, yes. I guess that doesn't preclude us using it for non-public things in the future, although I can't currently think of any scenarios where we'd want to do that.

Reply:
Great, I'll take care of this then.